core/src/main/java/org/apache/calcite/sql/dialect/HiveSqlDialect.java

```diff
@@ -678,5 +682,26 @@ private SqlCharStringLiteral creteDateTimeFormatSqlCharLiteral(String format) {
     return super.getDateTimeFormatString(standardDateFormat, dateTimeFormatMap);
   }
 
+  private void unparsePlusMinus(SqlWriter writer, SqlCall call, int leftPrec, int rightPrec) {
+    switch (call.getOperator().getName()) {
+    case "TIMESTAMP_ADD":
+    case "TIMESTAMP_SUB":
+      call.operand(0).unparse(writer, leftPrec, rightPrec);
```

Comment:
do we need to add operator before unparsing?

Reply:
no, we don't need to add the operator in hive and spark